megamek/src/megamek/client/ui/swing/tooltip/UnitToolTip.java

```diff
@@ -98,12 +101,12 @@ private static StringBuilder getEntityTipTable(Entity entity, Player localPlayer
             }
             result.append(inGameValues(entity, localPlayer));
             if (pilotInfo) {
-                result.append(PilotToolTip.getPilotTipShort(entity,
-                        GUIPreferences.getInstance().getBoolean(GUIPreferences.SHOW_PILOT_PORTRAIT_TT)));
+                result.append(PilotToolTip.getPilotTipShort(entity, GUIP.getBoolean(GUIPreferences.SHOW_PILOT_PORTRAIT_TT)));
             } else {
                 result.append("<BR>");
             }
         }
+        result.append("\n");

```

Comment:
Why are all these \n added?

Reply:
just makes it easier to see if i did things correctly, can remove if needed